Lab3/Weather/index.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" type="text/css" href="src/css/index.css"/>
+    <link rel="stylesheet" type="text/css" href="src/css/find.css"/>
+    <link rel="stylesheet" type="text/css" href="src/css/navbar.css"/>
+    <link rel="stylesheet" type="text/css" href="src/css/Temperature.css"/>
+    <link rel="stylesheet" type="text/css" href="src/css/MainPressure.css"/>
+    <link rel="stylesheet" type="text/css" href="src/css/MainHumidity.css"/>
+    <link rel="stylesheet" type="text/css" href="src/css/NewWind.css"/>
+    <link rel="stylesheet" type="text/css" href="src/css/Wind.css"/>
+    <title>Title</title>
+</head>
+<body>
+<header>
+    <nav class="dws-menu">
+        <div class="logo"><a href="index.html"><img src="src/img/weblogo.jpg"  height="40" width="150"></a></div>
+        <ul>
+            <li><a href="forecast.html">3-day forecast</a></li>
+            <li><a href="Map.html">Map</a></li>
+            <li><a href="#"></i>Coming Soon</a></li>
+            <li><a href="#"></i>More</a>
+                <ul>
+                    <li><a href="#">Account</a></li>
+                    <li><a href="#">About</a></li>
+                </ul>
+            </li>
+        </ul>
+    </nav>
+</header>
+<section class="wrapper-main">
+    <section class="upper">
+        <section class="temperature">
+
+            <article class="widget">
+                <div class="time">
+                    <h6>12:12</h6></div>
+                <div class="weatherIcon"></div>
+                <div class="weatherData">
+                    <h1 class="temperature-1">25&deg;</h1>
+                    <h2 class="description">Partly Cloudy</h2>
```

Comment:
tags like `<h1..6>` should be used only like headers for page or a section. Don't use them to represent a set of information. Use `<p>`